content/doc/book/updating/windows.adoc

```diff
@@ -0,0 +1,72 @@
+---
+layout: section
+title: Windows
+---
+ifdef::backend-html5[]
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc: left
+:toclevels: 3
+:hide-uri-scheme:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+endif::[]
+
+include::doc/book/updating/_pre-upgrade-wizard.adoc[]
+
+== Upgrade steps on Windows
+
+There are two ways to upgrade Jenkins in Windows:
```

Comment:
Needs instructions to guide the user to upgrade with the same technique they used for installation.  If they installed with MSI, they should upgrade with MSI.  If they installed by copying the war file, they should upgrade by replacing the war file.

Reply:
Thanks for the suggestions @MarkEWaite I'll add the instructions